docs/public/operators-on-minikube.adoc

```diff
@@ -0,0 +1,204 @@
+== Installing Operators on minikube
+
+This guide assumes that you are using link:https://minikube.sigs.k8s.io/docs/[minikube] v1.11.0 or newer.
+
+In this guide, we will discuss installing two Operators on a minikube environment:
+
+. etcd Operator
```

Comment:
why etcd? Shouldn't this be a little bit more generic? I don't expect that a lot of odo users will use etcd.

Reply:
Because our example on the [odo.dev](https://odo.dev/docs/operator-hub/#linking-an-odo-component-with-an-operator-backed-service) website explains the process using an etcd Operator.